defaults/main.yml

```diff
@@ -1,6 +1,15 @@
 # true if IPv6 is needed
 network_ipv6_enable: false              # sshd + ssh
 
+# true if sshd should be started and enabled
+ssh_server_enabled: false               # sshd
+
+# true if DNS resolutions are needed
+ssh_use_dns: true                       # sshd
```

Comment:
As this does not really have any security benefits, the default `no` should be set here.